cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/strategies/index.js

```diff
@@ -31,14 +31,14 @@ function Strategies() {
 
   return (
     <section className="strategies">
-      <header className="strategies-header">
+      {/* <header className="strategies-header">
```

Comment:
Any reason for leaving this in as a comment?

Reply:
Yes, this is the comment that I want to leave in for now.  I am going to show the client the screen without this section and make sure that they want it like that.